doc/api/tools-messagev.rst

```diff
@@ -0,0 +1,54 @@
+MESSAGE V input files
+*********************
+
+This document describes some of the file formats for the pre-:mod:`.ixmp` MESSAGE model, a.k.a. **MESSAGE V**, and code in :mod:`message_ix_models.tools.messagev` that reads these formats.
+
+.. note:: See also the earlier :doc:`import_from_msgV_sqlite` for similar code/descriptions.
```

Comment:
This `:doc:` reference does not seem to build a link in the [final docs page](https://iiasa-energy-program-message-ix--208.com.readthedocs.build/projects/models/en/208/api/tools-messagev.html).

Reply:
The referenced file has not been migrated from message_data, and perhaps never will. But noted; I will open a follow-up issue to clean up this docs page.